scripts/build.js

```diff
@@ -31,30 +31,24 @@ build({
   bundle: true,
   target: 'es6',
   outfile: 'dist/LmcCookieConsentManager.cjs',
-  platform: 'node',
+  platform: 'browser',
   format: 'cjs',
   /**
    * the "main" field was ignored (main fields must be configured manually when using the "neutral" platform)
    * because vanilla-cookie-consent is set as `main` in package.json
    */
   mainFields: ['main'],
-})
-  .then(() => {
-    const options = {
-      files: './dist/LmcCookieConsentManager.cjs',
-      from: /0 && \(module.exports = {}\);/gm,
-      to: 'module.exports = LmcCookieConsentManager_default;',
-    };
-
-    replace(options)
-      .then((results) => {
-        console.log('Replacement results:', results);
-      })
-      .catch((error) => {
-        console.error('Error occurred:', error);
-      });
-  })
-  .catch((error) => {
-    console.error(error);
-    process.exit(1);
-  });
+}).then(() => {
+  // Annotate the CommonJS export names for ESM import in node
+  fs.appendFile(
+    './dist/LmcCookieConsentManager.cjs',
+    `// Annotate the CommonJS export names for ESM import in node:
+module.exports = LmcCookieConsentManager_default`,
```

Comment:
```suggestion
module.exports = LmcCookieConsentManager_default;
`,
```

(semicolon + newline on the end of the file)